src/main/java/wedlog/address/model/AddressBook.java

```diff
@@ -159,6 +159,15 @@ public void removeGuest(Guest key) {
         guests.remove(key);
     }
 
+    /**
+     * Returns a {@code RsvpStatistics} with information on the RSVP statuses of guests
+     * @return {@code RsvpStatistics}
+     */
+    public RsvpStatistics getRsvpStatistics() {
+        return new RsvpStatistics(guests.getNumGuestsRsvpYes(), guests.getNumGuestsRsvpNo(),
```

Comment:
Perhaps it would be better to implement the logic of getting RSVP statistics within UniqueGuestList? would also eliminate the need for getters.